core/fetcher/fetcher.go

```diff
@@ -142,16 +142,17 @@ func (f *Fetcher) fetchProposerData(ctx context.Context, slot int64, argSet core
 			Slot: slot,
 			Type: core.DutyRandao,
 		}
-		randao, err := f.aggSigDBFunc(ctx, dutyRandao, pubkey)
+		signedData, err := f.aggSigDBFunc(ctx, dutyRandao, pubkey)
```

Comment:
maybe rename it to `randaoData` rather naming it same as return type of the function